domain/src/main/assets/test_prototype_exploration.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
See https://github.com/oppia/oppia-android/pull/1067#discussion_r427805105. Please resolve that, specifically:
- Rename this JSON file
- Update its title & ID to correspond to be drag-and-drop specific
- Update StateRetrieverTest to support loading different explorations by having it take an exploration name as input when creating the State object

Reply:
DONE